docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -430,3 +437,75 @@ export default clerkMiddleware(
 
 > [!NOTE]
 > Dynamic keys are not accessible on the client-side.
+
+### `OrganizationSyncOptions`
+
+The `organizationSyncOptions` property on the [`clerkMiddleware()`](#clerk-middleware-options) options
+object has the type `OrganizationSyncOptions`, which has the following properties:
+
+<Properties>
+  - `organizationPatterns`
+  - <code>[Pattern](#pattern)\[]</code>
+
+  Specifies URL patterns that are organization-specific, containing an organization ID or slug as a path parameter. If a request
+  matches this path, the organization identifier will be extracted and activated before rendering.
```

Comment:
when you say "the organization identifier will be activated", do you mean the organization identifier will be used to set that org as active?

Reply:
Yes, and I like your wording better!

e26600da08e4000e86654a273c62e36e14dd95c1